src/test/test.js

```diff
@@ -411,9 +429,55 @@ describe('Registrar', () => {
     });
   });
 
-  describe.skip('#invalidateName()', () => {
+  describe('#invalidateName()', () => {
     it('Should do something specific', (done) => {
-      done();
+      const hash = registrar.sha3('foo');
+      registrar.contract.startAuctionsAndBid([hash], shortBid.shaBid, 
+        { from: accounts[0], gas: 4700000, value: web3.toWei(1, 'ether')}, 
+        (startAndBidErr, startAndBidResult) => {
+          assert.equal(startAndBidErr, null); // this is failing for some reason with "invalid opcode"
+          assert.ok(startAndBidResult != null);
+          // fast forward to reveal period
+          web3.currentProvider.sendAsync({
+            jsonrpc: '2.0',
+            method: 'evm_increaseTime',
+            params: [3 * 86400],  // 86400 seconds in a day
+            id: Date.now()
+          }, () => {
+            timeDiff += 3 * 86400;
+            // unseal the bid
+            registrar.unsealBid(shortBid, { from: accounts[0], gas: 4700000 }, (unsealErr, unsealResult) => {
+              assert.equal(unsealErr, null);
+              assert.ok(unsealResult != null);
+              // fast forward to registration date
+              web3.currentProvider.sendAsync({
+                jsonrpc: '2.0',
+                method: 'evm_increaseTime',
+                params: [2 * 86400],  // 86400 seconds in a day
+                id: Date.now()
+              }, () => {
+                timeDiff += 8 * 7 * 86400;
+                registrar.getEntry('foo', (entryErr1, entryResult1) => {
+                  assert.equal(entryErr1, null);
+                  assert.equal(entryResult1.name, 'foo');
+                  assert.ok(entryResult1.deed.owner.slice(0,2) == '0x');
+                  assert.equal(entryResult1.mode, 'forbidden-can-invalidate');
+                  registrar.invalidateName('foo', { from: accounts[1], gas: 4700000 }, (invalidateErr, invalidateResult) => {
```

Comment:
@alexvandesande I've added the `registrar.invalidateName` method to the lib, but for whatever reason I'm unable to get this test to pass. It's returning an invalid opcode error. Appreciate if you can take a look.